.github/ISSUE_TEMPLATE/feature.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Feature
+about: Describe this issue template's purpose here.
+title: ''
+labels: ''
+assignees: ''
+
```

Comment:
I think some of this might be duplicating info already in the PR--The Name: is in the title of the PR, About: is in the General Description (a section below), Labels are added to the PR on the right of the PR, Assignees are also attached to the PR. Is it ok with you if we don't duplicate info on the PR that is on the PR itself or in the template below? Copy-pasting might lead to conflict if one changes and we don't manually update the PR?